### DISCUSSION TITLE
When is a next LTS after 92.0 is planned?

### DISCUSSION THREAD

**Discussion by nirGNU** [avocado-framework/avocado#5711] [Q&A] [answered]
Hi @clebergnu and @richtja 

When is a new LTS release planned? 
There are some really important fixes made after 92.0 which are required to be deployed in the environment here.  

Requesting to please provide an LTS based on the latest code :) 

Thanks,
Nir.

**Comment by richtja** [comment, marked as answer]
Hi @nirGNU, right now we run pre-release checks and we will release the avocado 102 very soon. After the release we will work on stabilization tasks like bug fix, usability, documentation and extended testing etc. and then we will release 103 which will be LTS version. So long story short, you can expect LTS version in release 103 which should be released between July and August.

**Comment by nirGNU** [comment]
103 LTS is released hence closed this discussion.
